### PR TITLE
Fixes openapi for new Auction

### PR DESCRIPTION
**Pull request by sunce86**
# Description
Follow up for https://github.com/cowprotocol/services/pull/2246

now the `/api/v1/auction` refers to the `AuctionOrder` which has it's own list of fields.

### CODE REVIEW

File: crates/orderbook/openapi.yml [hunk 1]
```diff
@@ -1448,3 +1557,39 @@ components:
         totalSurplus:
           type: string
           description: The total surplus.
+    InteractionData:
+      type: object
+      properties:
+        target:
+          $ref: "#/components/schemas/Address"
+        value:
+          type: number
```
Comment (MartinquaXD) [inline crates/orderbook/openapi.yml:1566]: Since this is a `U256` it should probably be a decimal string.

**Review by fleupold — APPROVED** [review]
We should really find a way to autogenerate those 🙈

**Comment by MartinquaXD** [comment]
> We should really find a way to autogenerate those 🙈

https://github.com/cowprotocol/services/issues/2203

File: crates/orderbook/openapi.yml (hunk 1 above)
Comment (sunce86) [reply crates/orderbook/openapi.yml]: Right, fixed!

**Pull request merged**